### PR TITLE
[15.0][MIG] website_event_crm > website_event_crm_invitation: Migration to 15.0

### PR DESCRIPTION
**Pull request by chienandalu**
- Renaming of the module to avoid colission with the new core module of the same name (from v14) 
- Migration adaptations

cc @Tecnativa TT45105

please review @pedrobaeza @pilarvargas-tecnativa

### CODE REVIEW

File: website_event_crm_invitation/data/mail_template_data.xml [hunk 1]
```diff
@@ -5,20 +5,20 @@
     <record id="crm_lead_event_type_tpl" model="mail.template">
         <field name="name">Invite to event category</field>
         <field name="auto_delete" eval="False" />
-        <field name="email_to">${object.email_from or ''}</field>
-        <field name="partner_to">${object.partner_id.id or ''}</field>
+        <field name="email_to">{{object.email_from or ''}}</field>
```
Comment (pedrobaeza) [inline website_event_crm_invitation/data/mail_template_data.xml:8]: This requires a migration script.

**Review by pedrobaeza — COMMENTED** [review]
OpenUpgrade should contain the renaming as well.

**Comment by chienandalu** [comment]
> OpenUpgrade should contain the renaming as well.

Should I do it in v14 or directly in v15?

**Comment by pedrobaeza** [comment]
As `website_event_crm` exists already in 14.0: https://github.com/odoo/odoo/tree/14.0/addons/website_event_crm, it should be in OU 14

**Review by pedrobaeza — APPROVED** [review]
Remember about the migration script for noupdate data.

**Comment by chienandalu** [comment]
> Remember about the migration script for noupdate data.

Yes, I'm working on it. I'm going to test it

**Comment by MiquelRForgeFlow** [comment]
Will this module be backported to v14?

**Comment by pedrobaeza** [comment]
Not planned for now.

**Review by pilarvargas-tecnativa — APPROVED** [review]
Functionally reviewed, thanks

**Pull request merged**